### ISSUE TITLE
Nothing happens when pressing on the add button.

### ISSUE THREAD

**Issue by mooksz** [felipeelia/cf7-repeatable-fields#31]
Hallo,

I just installed the plugin v.1.1.3 with CF7 v.5.1.7 on a wordpress v.5.3.2. Created a contact form with the following: 
`
[field_group emails id="emails-groups" tabindex:1]
[text text-315]
[/field_group]
`

When I ouput the shortcode the form is displaying and I can see the add and remove buttons. The only problem is when I press the add/remove button noting happens.

**Comment by felipeelia** [comment]
Hi @mooksz , can you see any errors in the Developer Tools Console?

**Comment by mooksz** [comment]
Unfortunately not, for some reason when I used the plugin on another website i had no issues.. Can't seem to find any errors in any logs either..

**Comment by jaumearagay** [comment]
I had the same issue having the "Contact Form 7 - PayPal & Stripe Add-on" also active. In the "Other" tab, "Redirect Method" was set to 2 by default and it was disabling Contact Form 7's JavaScript. Setting the redirect method to 1 reenabled the Contact Form 7's JavaScript and then when only the first set of repeatable fields were on screen I only had the "+" button available and after adding a second set of fields the "-" button was available too and working properly.

TL;DR: **Check if other plugins disable the Contact Form 7's JavaScript**! ;)
